### PR TITLE
Refresh Making GPG Keys page

### PR DESCRIPTION
**Pull request by slawekjaranowski**

### CODE REVIEW

File: content/apt/developers/release/pmc-gpg-keys.apt [hunk 1]
```diff
@@ -28,146 +28,175 @@
...
+ Public Keys used for signing Maven core, plugins and shared components are available for users at:\
+ {{https://downloads.apache.org/maven/KEYS}}
+
+ You need edit a file and follow provided instructions in SVN at:\
+ {{https://svn.apache.org/repos/asf/maven/project/KEYS}}
+
+* Distributing Your Public Keys
+
+ Your Public Keys <<MUST>> be available at public key server,
+ you can use one or evan all of currently common used key server
```
Comment (hgschmie) [inline content/apt/developers/release/pmc-gpg-keys.apt:48]: typo: even

File: content/apt/developers/release/pmc-gpg-keys.apt [hunk 2]
```diff
@@ -28,146 +28,175 @@
...
-sub   2048g/D2814A59 2006-10-10
+ All new <<RSA>> keys generated should be at least <<4096>> bits. Do not generate new DSA keys.
+
+* Maven Project Keys
+
+ Public Keys used for signing Maven core, plugins and shared components are available for users at:\
+ {{https://downloads.apache.org/maven/KEYS}}
+
+ You need edit a file and follow provided instructions in SVN at:\
+ {{https://svn.apache.org/repos/asf/maven/project/KEYS}}
```
Comment (hgschmie) [inline content/apt/developers/release/pmc-gpg-keys.apt:43]: the instructions on that file are unclear. Adding the key can be done by any committer but exposing them (moving to dist/release/maven) is a PMC activity. The document can be read (... I read it this way ...) that the whole thing (adding to the KEYS file etc.) is a PMC activity.

File: content/apt/developers/release/pmc-gpg-keys.apt (hunk 1 above)
Comment (hgschmie) [reply content/apt/developers/release/pmc-gpg-keys.apt:48]: from what I can see, the repository.apache.org server only uses keys.openpgp.org as the verification source.

File: content/apt/developers/release/pmc-gpg-keys.apt [hunk 3]
```diff
@@ -28,146 +28,175 @@
...
+  under certain conditions. See the file COPYING for details.
+
+  gpg: keyring `C:/Documents and Settings/Siveton Vincent/Application Data/gnupg\secring.gpg'
+  created
+  gpg: keyring `C:/Documents and Settings/Siveton Vincent/Application Data/gnupg\pubring.gpg'
+  created
+  Please select what kind of key you want:
+     (1) DSA and Elgamal (default)
+     (2) DSA (sign only)
+     (5) RSA (sign only)
```
Comment (hgschmie) [inline content/apt/developers/release/pmc-gpg-keys.apt:82]: given that above says "only use RSA with 4096 bits", the whole example is weird/wrong.

**Review by hgschmie — APPROVED** [review]

File: content/apt/developers/release/pmc-gpg-keys.apt (hunk 1 above)
Comment (slawekjaranowski) [reply content/apt/developers/release/pmc-gpg-keys.apt:48]: But other tools can use more servers

File: content/apt/developers/release/pmc-gpg-keys.apt (hunk 3 above)
Comment (slawekjaranowski) [reply content/apt/developers/release/pmc-gpg-keys.apt:82]: you are right - example is very old

File: content/apt/developers/release/pmc-gpg-keys.apt (hunk 2 above)
Comment (slawekjaranowski) [reply content/apt/developers/release/pmc-gpg-keys.apt]: In such case we can update instruction in KEYS file

**Comment by slawekjaranowski** [comment]
old outdated instruction was removed - we can point to ASF infra ... do work once

**Review by cstamas — APPROVED** [review]

**Review by olamy — APPROVED** [review]

**Review by slachiewicz — APPROVED** [review]

**Pull request merged**